pkg/tanka/export.go

```diff
@@ -38,7 +38,7 @@ type ExportEnvOpts struct {
 	// optional: only export specified Kubernetes manifests
 	Targets []string
 	// optional: options for parsing Environments
-	ParseParallelOpts ParseParallelOpts
+	ParallelOpts ParallelOpts
```

Comment:
Imo whether something uses Parallel underneath should not need to concern the consumer. Better include the fields you actually need top level in the `ExportOpts` struct

Also `ExportOpts` should embed `Opts` like all other opts types for consistency

Reply:
Ok, so you noticed it is used in Export, can we refactor that in another PR?